pipeline_utils/pipeline_deploy.py

```diff
@@ -0,0 +1,406 @@
+#!/usr/bin/env python3
+
+################################################
+#
+#   pipeline_deploy, from YAML format
+#
+#   Michele Berselli - berselli.michele@gmail.com
+#
+################################################
+
+import os, sys, subprocess
+import json
+import glob
+import boto3
+import structlog
+from dcicutils import ff_utils, s3_utils
+from pipeline_utils.lib import yaml_parser
+
+
+###############################################################
+#   REPOSITORY
+#
+### Expected structure for the pipeline repository
+#
+#   pipeline
+#       wfl
+#       dockerfiles
+#            <image>
+#                Dockerfile
+#       portal_objects
+#           workflows
+#               <wfl>.yaml
+#           metaworkflows
+#               <mwfl>.yaml
+#           file_format.yaml
+#           file_reference.yaml
+#           software.yaml
+#       PIPELINE
+#       VERSION
+#
+###############################################################
+
+
+###############################################################
+#   Logger
+###############################################################
+logger = structlog.getLogger(__name__)
+
+
+###############################################################
+#   PostPatchRepo, class definition
+###############################################################
+class PostPatchRepo(object):
+
+
+    def __init__(self, args, repo, version='VERSION', pipeline='PIPELINE'):
+        """
+        """
+        # Init credentials
+        self.ff_key = None
+        self.kms_key_id = None
+        self.repo = repo
+        self.object_ = {
+            'Software': yaml_parser.YAMLSoftware,
+            'FileFormat': yaml_parser.YAMLFileFormat,
+            'FileReference': yaml_parser.YAMLFileReference,
+            'Workflow': yaml_parser.YAMLWorkflow,
+            'MetaWorkflow': yaml_parser.YAMLMetaWorkflow
+        }
+        self.filepath = {
+            'Software': 'portal_objects/software.yaml',
+            'FileFormat': 'portal_objects/file_format.yaml',
+            'FileReference': 'portal_objects/file_reference.yaml',
+            'Workflow': 'portal_objects/workflows',
+            'MetaWorkflow': 'portal_objects/metaworkflows',
```

Comment:
Should these be like the above?
```suggestion
            'Workflow': 'portal_objects/workflow.yaml',
            'MetaWorkflow': 'portal_objects/metaworkflow.yaml',
```

Reply:
this is pointing to folders